backend/lib/migrations/followups/to-v5.ts

```diff
@@ -0,0 +1,12 @@
+const VERSION = 5
+
+export default {
+  apply(followup) {
+    followup.surveys = followup.surveys.map((survey) => {
+      survey.touchedAts = [survey.openedAt]
+      return survey
+    })
```

Comment:
J'imagine qu'on peut supprimer le openedAt si il est remplacé par ce nouveau tableau
```suggestion
    followup.surveys = followup.surveys.map((survey) => {
      survey.touchedAts = [survey.openedAt]
      survey.openedAt = undefined
      return survey
    })
```